config/config_embed.go

```diff
@@ -0,0 +1,19 @@
+package config
+
+import (
+	_ "embed"
+)
+
+// Embedding configuration files
+//
+//go:embed protocol.mainnet.yml
```

Comment:
Have you tried to put it inside `cli/options`?

Reply:
yes. It then needs a relative path but `Path must not contain '.' or '..' path elements nor begin with a leading slash`

Reply:
Doesn't seem perfect, but looks like there are no other options. Hard links aren't compatible with git, symbolic links are prohibited by embed. The only other option is copy/autosync.

Reply:
> The only other option is copy/autosync.

I don't quite like duplication, thus I think we're OK with this file being placed inside `neo-go/config`.